src/deploy/functions/release/fabricator.ts

```diff
@@ -125,6 +125,10 @@ export class Fabricator {
       this.logOpStart("creating", endpoint);
       upserts.push(handle("create", endpoint, () => this.createEndpoint(endpoint, scraper)));
     }
+    for (const endpoint of changes.endpointsToSkip) {
+      this.logOpStart("skipping", endpoint);
+      upserts.push(handle("skip", endpoint, () => this.skipEndpoint()));
```

Comment:
Doesn't this lead to kind of a funny deploy log like:

```
Skipping function 1...
Finished skipping function...
```

It's not a total blocker, but I'd expect something simpler like:

```
Not deploying function 1 - no change since last deploy (hash=???) 
Not deploying function 2 - no change since last deploy (hash=???) 
```

Or even something more general like:

```
Skipping deployment of following functions since CLI detected no changes:
  - Function 1 (hash=?)
  - Function 2 (hash=?)

To force deploy these function, run command XXX 
```

Reply:
Updated so messaging is closer to that!